SAMtags.tex

```diff
@@ -55,7 +56,7 @@ \section{Standard tags}
   \hline
   {\bf Tag} & {\bf Type} & {\bf Description} \\
   \hline
-  {\tt AM} & i & The smallest template-independent mapping quality of segments in the rest \\
+  {\tt AM} & i & The smallest template-independent mapping quality in the template \\
```

Comment:
this look like a change in the definition. I've never used AM, but the reading of the original definition seems to imply that it does not take the mapping quality of this record into account, while the new definition does. Am I wrong about my interpretation? is this correction justified?